aries-site/src/pages/components/layer/center-layer.mdx

```diff
@@ -0,0 +1,97 @@
+
+<Example plain>
+  <CenterLayerAnatomy informational />
+</Example>
+
+| Label | Region           | Purpose           |                  Required                   | Notes                 |
+| ----- | ---------------- | ----------------- | :-----------------------------------------: | --------------------- |
+| **1** | **Close button** | Closes the layer. | <Checkmark a11yTitle="true" size="small" /> | Use "FormClose" icon. |
+
+The user should be able to dismiss the informational layer by clicking outside, `onClickOutside`, or using the Escape key, `onEsc`.
+
+### Actionable
+
+For layers that require user actions, place a "Cancel" action alongside the other footer actions. In some cases, a label other than "Cancel" may be appropriate.
+
+<Example plain>
+  <CenterLayerAnatomy />
+</Example>
+
+| Label | Region             | Purpose                                                          |                  Required                   | Notes                                                                                                                                                                                                                                |
+| ----- | ------------------ | ---------------------------------------------------------------- | :-----------------------------------------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **1** | **Footer actions** | Allows the user to confirm, apply, or cancel the layer's action. | <Checkmark a11yTitle="true" size="small" /> | The actions should be right-aligned with primary action on the right. Right-alignment signals progress and forward momentum. One action should be a "Cancel" button that closes the layer, returning the user to the previous state. |
+
+To prevent the user from losing progress due to accidental clicks outside of a layer, `onClickOutside` functionality should not be enabled. The user should be able to dismiss the actionable layer by using the Escape key, `onEsc`.
+
+### Double confirmations
+
+When data has been edited or changed and the user tries to close the layer, a double confirmation should be shown. This helps avoid unintentional loss of work.
+
+Learn more about the anatomy and implementation of a [double confirmation](/components/layer#double-confirmations) on the main layer page since this behavior applies to all layer types.
+
+## Examples
+
+### Confirming next steps
+
+<Example
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/layer/CenterInformational.js"
+  github="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/layer/CenterInformational.js"
```

Comment:
Does this url need to be different? e.g. https://github.com/grommet/hpe-design-system/blob/master/aries-site/src/examples/ instead of raw.github ....

Reply:
Done.